src/OrbitGl/App.cpp

```diff
@@ -366,6 +366,7 @@ OrbitApp::OrbitApp(orbit_gl::MainWindowInterface* main_window,
 OrbitApp::~OrbitApp() {
   AbortCapture();
   RequestSymbolDownloadStop(module_manager_->GetAllModuleData(), false);
+  thread_pool_->ShutdownAndWait();
```

Comment:
With this we are not fixing crashes that are in the middle of destructing OrbitApp, am I wrong?

Reply:
The dtor runs before the destruction of the member variables and this will/might hopefully fix some things.

Reply:
What Daniel said. In other words: we are crashes that are in the middle of destructing OrbitApp (hopefully)